### ISSUE TITLE
Kubernetes v1beta3 is missing ConfigMapKeyRef

### ISSUE THREAD

**Issue by mfojtik** [openshift/origin#7582]
It seems like this field was lost in rebase and cause this error when we run conversions:

```
[root@localhost origin]# ./hack/update-generated-conversions.sh
Generating for version v1beta3
E0224 11:39:46.995981   17177 conversion.go:71] error while generating conversion functions for v1beta3.BuildRequest: errors: <nil>, couldn't find a corresponding field ConfigMapKeyRef in v1beta3.EnvVarSource
E0224 11:39:47.023649   17177 conversion.go:71] error while generating conversion functions for v1beta3.DeploymentConfig: errors: <nil>, couldn't find a corresponding field ConfigMapKeyRef in v1beta3.EnvVarSource
E0224 11:39:47.129912   17177 conversion.go:71] error while generating conversion functions for v1beta3.DeploymentConfigList: errors: <nil>, couldn't find a corresponding field ConfigMapKeyRef in v1beta3.EnvVarSource
Generating for version v1
[root@localhost origin]#
```

(that field does not exists in k8s v1beta3)

**Comment by mfojtik** [comment]
@soltysh @kargakis FYI

**Comment by liggitt** [comment]
This didn't exist when v1beta3 was supported. I think we can skip it

**Comment by mfojtik** [comment]
@liggitt does it break conversion generation for v1beta3 objects that has EnvVar ?

**Comment by kargakis** [comment]
How come our serialization tests don't complain already

**Comment by deads2k** [comment]
> How come our serialization tests don't complain already

v1beta3 fuzzing tests were removed in a previous rebase after the point at which we no longer support roundtripping to and from them.  Recall that we stopped supporting storing in v1beta3 back in the 1.1.0 timeframe.  After that, we only need readability from v1beta3.  There's some argument for a different kind of test for taking a previously valid v1beta3, defaulting it, converting it, and getting a valid object back out, but haven't built such a thing.
